src/main/java/fr/xephi/authme/datasource/SQLite.java

```diff
@@ -467,6 +467,48 @@ public void setUnlogged(String user) {
     }
```

Comment:
Like in the MySQL class, here I'm missing the initialization of the `hasSession` column